variables.tf

```diff
@@ -67,7 +73,7 @@ variable "dns_zone_id" {
 
 variable "elasticsearch_version" {
   type        = string
-  default     = "6.5"
```

Comment:
Let's leave the default at 6.5 and override for the v7 deployments

Reply:
yeah I agree at least for now until everthing is changed over

Reply:
done.